### PR TITLE
.github/ISSUE_TEMPLATE/bug.yml: update syntax.

### PR DESCRIPTION
**Pull request by MikeMcQuaid**
This syntax is changing today and needs fixed CC @deniseyu and @Oneill38 for review.

### CODE REVIEW

**Review by SMillerDev — DISMISSED** [review]

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 1]
```diff
@@ -3,12 +3,13 @@ about: "If you're sure it's reproducible and not just your machine: submit an is
 labels: bug
 issue_body: false
 inputs:
```
Comment (Oneill38) [inline .github/ISSUE_TEMPLATE/bug.yml:5]:
```suggestion
body:
```

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 2]
```diff
@@ -3,12 +3,13 @@ about: "If you're sure it's reproducible and not just your machine: submit an is
...
+  - type: markdown
     attributes:
       value: Please note we will close your issue without comment if you do not fill out the issue checklist below and provide ALL the requested information. If you repeatedly fail to use the issue template, we will block you from ever submitting issues to Homebrew again.
   - type: textarea
     attributes:
       label: "`brew gist-logs <formula>` link OR `brew config` AND `brew doctor` output"
+    validations:
       required: true
   - type: checkboxes
     attributes:
```
Comment (Oneill38) [inline .github/ISSUE_TEMPLATE/bug.yml:15]: for some reason I can't click on the specific line to make the code suggestion but `choices` here should be changed to `options`

Comment (MikeMcQuaid) [reply .github/ISSUE_TEMPLATE/bug.yml:15]: Fixed, thanks @Oneill38!

**Review by Oneill38 — APPROVED** [review]
✨

**Review by SMillerDev — APPROVED** [review]

**Pull request merged**